tests/unit/features/StringFeatures_unittest.cc

```diff
@@ -81,9 +81,11 @@ TEST(StringFeaturesTest,copy_subset)
 TEST(StringFeaturesTest,clone)
```

Comment:
This test is kinda bad. Maybe we should just delete it and work on the non-empty tests for all sgobjects instead?

Reply:
It is a good idea, however, let's do that in another PR?

Reply:
yep